graphql-java-spring-webflux/src/main/java/graphql/spring/web/reactive/GraphQLContextBuilder.java

```diff
@@ -0,0 +1,10 @@
+package graphql.spring.web.reactive;
+
+import graphql.PublicApi;
+import org.springframework.web.server.ServerWebExchange;
+
+@PublicApi
+public interface GraphQLContextBuilder {
+
+    Object build(ServerWebExchange webRequest);
```

Comment:
At the risk of making "InvocationData" API, would we not want that here since its parsed and ready to go

     Object build(GraphQLInvocationData invocationData, ServerWebExchange webRequest);

 or perhaps it should be

   Object build(ExecutionInput startingExecutionInput, ServerWebExchange webRequest);

OR perhaps we don't have this but rather a `ExecutionInputCustomizer` where they can return the starting `ExecutionInput` or they can `ei.transform()` it themselves.